issue with scm-bitbucket.

while deploying scm-bitbucket i am encountering the below issue.

scm-bitbucket-collector/target/bitbucket-scm-collector-3.1.1-SNAPSHOT.jar!/lib/aopalliance-1.0.jar!/, jar:file:/opt/hygieia-scm-bitbucket-collector/target/bitbucket-scm-collector-3.1.1-SNAPSHOT.jar!/lib/spring-beans-4.2.5.RELEASE.jar!/, jar:file:/opt/hygieia-scm-bitbucket-collector/target/bitbucket-scm-collector-3.1.1-SNAPSHOT.jar!/lib/spring-context-4.2.5.RELEASE.jar!/]
2019-09-19 17:21:02,222 [main] ERROR o.s.boot.SpringApplication - Application startup failed
org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'metadataRepository': Invocation of init method failed; nested exception is org.springframework.dao.DataIntegrityViolationException: Cannot create index for '' in collection 'metadata' with keys '{ "key" : 1 , "type" : 1}' and options '{ "name" : "unique_mdata_key_type" , "unique" : true}'. Index already defined as '{ "v" : 2 , "key" : { "key" : 1 , "type" : 1} , "name" : "unique_mdata_key_type" , "ns" : "dashboarddb.metadata"}'.; nested exception is com.mongodb.MongoCommandException: Command failed with error 85: 'Index with name: unique_mdata_key_type already exists with different options' on server localhost:27017. The full response is { "ok" : 0.0, "errmsg" : "Index with name: unique_mdata_key_type already exists with different options", "code" : 85, "codeName" : "IndexOptionsConflict" }
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.initializeBean(AbstractAutowireCapableBeanFactory.java:1578) ~[spring-beans-4.2.5.RELEASE.jar!/:4.2.5.RELEASE]
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:545) ~[spring-beans-4.2.5.RELEASE.jar!/:4.2.5.RELEASE]
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:482) ~[spring-beans-4.2.5.RELEASE.jar!/:4.2.5.RELEASE]
	at org.springframework.beans.factory.support.AbstractBeanFactory$1.getObject(AbstractBeanFactory.java:306) ~[spring-beans-4.2.5.RELEASE.jar!/:4.2.5.RELEASE]
	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:230) ~[spring-beans-4.2.5.RELEASE.jar!/:4.2.5.RELEASE]
	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:302) ~[spring-beans-4.2.5.RELEASE.jar!/:4.2.5.RELEASE]
	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:197) ~[spring-beans-4.2.5.RELEASE.jar!/:4.2.5.RELEASE]
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.preInstantiateSingletons(DefaultListableBeanFactory.java:753) ~[spring-beans-4.2.5.RELEASE.jar!/:4.2.5.RELEASE]
	at org.springframework.context.support.AbstractApplicationContext.finishBeanFactoryInitialization(AbstractApplicationContext.java:839) ~[spring-context-4.2.5.RELEASE.jar!/:4.2.5.RELEASE]
	at org.springframework.context.support.AbstractApplicationContext.refresh(AbstractApplicationContext.java:538) ~[spring-context-4.2.5.RELEASE.jar!/:4.2.5.RELEASE]
	at org.springframework.boot.SpringApplication.refresh(SpringApplication.java:752) [spring-boot-1.3.0.RELEASE.jar!/:1.3.0.RELEASE]
	at org.springframework.boot.SpringApplication.doRun(SpringApplication.java:347) [spring-boot-1.3.0.RELEASE.jar!/:1.3.0.RELEASE]
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:295) [spring-boot-1.3.0.RELEASE.jar!/:1.3.0.RELEASE]
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:1112) [spring-boot-1.3.0.RELEASE.jar!/:1.3.0.RELEASE]
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:1101) [spring-boot-1.3.0.RELEASE.jar!/:1.3.0.RELEASE]
	at com.capitalone.dashboard.Application.main(Application.java:13) [bitbucket-scm-collector-3.1.1-SNAPSHOT.jar!/:3.1.1-SNAPSHOT]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[na:1.8.0_222]
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[na:1.8.0_222]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[na:1.8.0_222]
	at java.lang.reflect.Method.invoke(Method.java:498) ~[na:1.8.0_222]
	at org.springframework.boot.loader.MainMethodRunner.run(MainMethodRunner.java:53) [bitbucket-scm-collector-3.1.1-SNAPSHOT.jar!/:3.1.1-SNAPSHOT]
	at java.lang.Thread.run(Thread.java:748) [na:1.8.0_222]
Caused by: org.springframework.dao.DataIntegrityViolationException: Cannot create index for '' in collection 'metadata' with keys '{ "key" : 1 , "type" : 1}' and options '{ "name" : "unique_mdata_key_type" , "unique" : true}'. Index already defined as '{ "v" : 2 , "key" : { "key" : 1 , "type" : 1} , "name" : "unique_mdata_key_type" , "ns" : "dashboarddb.metadata"}'.; nested exception is com.mongodb.MongoCommandException: Command failed with error 85: 'Index with name: unique_mdata_key_type already exists with different options' on server localhost:27017. The full response is { "ok" : 0.0, "errmsg" : "Index with name: unique_mdata_key_type already exists with different options", "code" : 85, "codeName" : "IndexOptionsConflict" }

--- a/pom.xml
+++ b/pom.xml
@@ -51,7 +51,7 @@
     <repository.name>hygieia-scm-bitbucket-collector</repository.name>
     <apache.rat.plugin.version>0.13</apache.rat.plugin.version>
     <bc.version>3.0.1</bc.version>
-    <com.capitalone.dashboard.core.version>3.1.2</com.capitalone.dashboard.core.version>
+    <com.capitalone.dashboard.core.version>3.1.5</com.capitalone.dashboard.core.version>
     <commons.io.version>2.4</commons.io.version>
     <commons.lang.version>3.8.1</commons.lang.version>
     <coveralls.maven.plugin.version>4.3.0</coveralls.maven.plugin.version>

